### ISSUE TITLE
Incorrect display of minor ticks in Log Scale plot with large font size

### ISSUE THREAD

**Issue by billtau** [matplotlib/matplotlib#8419]
**Bug summary**

- If I use a larger font size than the default one, a large range of data in Log scale plot results in incorrect display of minor ticks.

**Code for reproduction**
```python
import matplotlib.pyplot as plt
import matplotlib as mpl

mpl.rcParams['font.size'] = 20

plt.close()

plt.plot([0,1], [1e-7, 1e-0])
plt.yscale('log')

plt.show()
```

**Actual outcome**

![log_wrong](https://cloud.githubusercontent.com/assets/14089622/24592748/530f54fe-184e-11e7-9d9a-d31aa6b63c3b.png)

**Expected outcome**

- I would expect the minor ticks to be not displayed at all, or displayed correctly like one with default font size.

![log_correct](https://cloud.githubusercontent.com/assets/14089622/24592757/688343c2-184e-11e7-8098-fa0ebabc9a37.png)

**Matplotlib version**
- Matplotlib version, Python version and Platform (Windows, OSX, Linux ...)

Matplotlib 2.0.0, Python 3.5.3, OSX

- How did you install Matplotlib and Python (pip, anaconda, from source ...)

via Macports

**Comment by phobson** [comment]
Thanks for the report. I *think* this has already been fixed in master via #8381

**Comment by efiring** [comment]
Correct.  Closing.
